scripts/resolve-version-lib.js

```diff
@@ -3,45 +3,44 @@ const path = require('path');
 const fs = require('fs');
 
 //=============================================================
-// UNIT TESTS: tools/script-tests/test/resolve-version.test.js 
+// UNIT TESTS: tools/script-tests/test/resolve-version.test.js
 //=============================================================
 
 function resolveVersion(rootdir) {
   const ALLOWED_RELEASE_TYPES = [ 'alpha', 'rc', 'stable' ];
   const MIN_MAJOR = 1, MAX_MAJOR = 2; // extra safety: update to allow new major versions
-  
+
   //
   // parse release.json
   //
   const releaseFile = path.join(rootdir, 'release.json');
   const releaseConfig = require(releaseFile);
   const majorVersion = releaseConfig.majorVersion;
   const releaseType = releaseConfig.releaseType;
-  const distTag = releaseConfig.distTag;
   if (!majorVersion) { throw new Error(`"majorVersion"" must be defined in ${releaseFile}`); }
   if (!releaseType) { throw new Error(`"releaseType" must be defined in ${releaseFile}`); }
   if (typeof(majorVersion) !== 'number') { throw new Error(`majorVersion=${majorVersion} must be a number`); }
   if (majorVersion < MIN_MAJOR || majorVersion > MAX_MAJOR) { throw new Error(`majorVersion=${majorVersion} is an unsupported major version (should be between ${MIN_MAJOR} and ${MAX_MAJOR})`); }
   if (!ALLOWED_RELEASE_TYPES.includes(releaseType)) { throw new Error(`releaseType=${releaseType} is not allowed. Allowed values: ${ALLOWED_RELEASE_TYPES.join(',')}`); }
-  
+
   //
   // resolve and check that we have a version file
   //
-  
+
   const versionFile = `version.v${majorVersion}.json`;
   const versionFilePath = path.join(rootdir, versionFile);
   if (!fs.existsSync(versionFilePath)) {
     throw new Error(`unable to find version file ${versionFile} for major version ${majorVersion}`);
   }
-  
+
   //
   // validate that current version matches the requirements
   //
-  
+
   const currentVersion = require(versionFilePath).version;
   console.error(`current version: ${currentVersion}`);
```

Comment:
minor: an opportunity to fix this annoyance from showing up in unit tests.

I would maybe put this behind a method like -

```ts
function log(msg) {
  if (process.env('DEBUG')) { console.error(msg); }
}
```

Reply:
Where _is_ this logging message currently being looked for? Does/will it set the DEBUG env? Adding something like the above without diving into the usages is akin to just deleting it. It's being used by pack, bump, check-api-compatibility, etc. I can dive into each and see which ones benefit from the logging, but I'd like to call that out of scope here.

Reply:
Fair enough. You could just drop it too.